lib/pc_util.c

```diff
@@ -263,3 +288,123 @@ void pc_bounds_merge(PCBOUNDS *b1, const PCBOUNDS *b2)
 	if ( b2->ymax > b1->ymax ) b1->ymax = b2->ymax;
 }
 
+static uint32_t srid_mask = 0x20000000;
+static uint32_t m_mask = 0x40000000;
+static uint32_t z_mask = 0x80000000;
+
+uint8_t *
+pc_bounds_to_bounding_diagonal_wkb(
```

Comment:
Could be changed to `pc_bounding_diagonal_wkb_from_bounds`. And the second function could be changed to `pc_bounding_diagonal_wkb_from_stats`.